app/controllers/start_group_controller.rb

```diff
@@ -44,6 +45,7 @@ def create_signed_out
       StartGroupService.invite_admin_to_group(group: @group,
                                               name: @name,
                                               email: @email)
+      @group.update_attribute(:is_referral, false)
```

Comment:
i.e. I think it is actually good to use true, false, and nil because I want a subgroup.is_referral to return nil